Stellarium/stellarium
# #100 news bookmarks functionalities

### Description
- sort bookmarks by column,
- possibilty to cretae multiple bookmarks lists.

Fixes #100 

Screenshots (if appropriate): no screenshot, but I can provide it if needed.

Type of change

### **How Has This Been Tested ?**
**Test Configuration:**
Operating system: 
 - Linux/Kubuntu 19.0
 - Intel Core i7-9700 8X 3,00 GHz / 16Go DDR4 / HDD: 480Go SSD + HDD: 1To
 - Graphics Card: GeForce RTX2060 6Go

I did many tests:
- creation of multiple bookmarks list,
- deletion of bookmarks list,
- check the loading lists when opening bookmarks windows,
- import/export of bookmarks in(from) a selected list,
- highlight bookmarks from a selected list/clear highlight,
- adding/deleted bookmark.

### **Checklist:**
  - [x] [checkbox:checked] I have performed a self-review of my own code,
